_posts/2024-05-27-gsoc-outreachy-students.md

```diff
@@ -0,0 +1,86 @@
+
+Lawson is a computer science student at Arizona State University (ASU) graduating in December 2024. You can find Lawson on GitHub as [@ljwoods2](https://github.com/ljwoods2) and on [LinkedIn](https://www.linkedin.com/in/lawson-woods/). To see updates on the `zarrtraj` project as well as his other projects you can check out [his blog](https://ljwoods2.github.io/).
+
+## Valerij Talagayev: [2D Visualization for Small Molecules](https://summerofcode.withgoogle.com/programs/2024/projects/sfy3kuqc)
+
+<img
+src="https://avatars.githubusercontent.com/talagayev"
+title="Valerij Talagayev" alt="Valerij Talagayev"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+2D Visualization of small molecules is quite a helpful way to recognize the different parts of the small molecule. The main objective of Valerij's GSoC project will revolve around the 2D visualization of small molecules through the application of the MDAnalysis RDKit converter, which would allow to convert ligands into RDKit objects and thus allow the user to look at the ligand in his MDAnalysis Universe in an easy to use interactive display in [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/).
+
+Valerij is a pharmacy student who graduated from the University of Marburg, where he worked at the [Kolb Lab](https://www.uni-marburg.de/en/fb16/ipc/kolb-group). Valerij is currently a PhD student at the Free University of Berlin in the [Wolber Lab](https://www.bcp.fu-berlin.de/en/pharmazie/faecher/pharmazeutische_chemie/wolber/index.html) with his research revolving around Toll-like receptor antagonists with addition to method development such as [OpenMMDL](https://github.com/wolberlab/OpenMMDL).
+
+You can find Valerij on Github as [talagayev](https://github.com/talagayev) and on [LinkedIn](https://www.linkedin.com/in/valerij-talagayev-260bb820b). To see the updates on the 2D visualization of small molecules project follow his [blog](https://talagayev.github.io/). 
+
+## Luna Morrow: [Extend MDAnalysis Interoperability with OpenBabel](https://summerofcode.withgoogle.com/programs/2024/projects/yLzX6MjS)
+
+<img
+src="https://avatars.githubusercontent.com/lunamorrow"
+title="Luna Morrow" alt="Luna Morrow"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Direct interoperability between molecular dynamics software is critical for enabling collaboration, data transfer and easy use by scientists. [OpenBabel](http://openbabel.org) is a popular toolbox for chemical molecular modelling as it enables searching, file conversions, basic analysis and data storage. The ability to interconvert between OpenBabel OBMols and MDAnalysis AtomGroups will enable input and writing of over 100 chemical data file formats.
```

Comment:
Very nitpicky suggestion: not sure if `data storage` makes much sense, so I'd rephrase it as such:
```suggestion
Direct interoperability between molecular dynamics software is critical for enabling collaboration, data transfer and easy use by scientists. [OpenBabel](http://openbabel.org) is a popular toolbox for chemical molecular modelling as it allows converting, searching, filtering and analyzing chemical data. The ability to interconvert between OpenBabel OBMols and MDAnalysis AtomGroups will enable input and writing of over 100 chemical data file formats.
```

Reply:
Pinging @lunamorrow to confirm approval of the change

Reply:
Are we able to say "[OpenBabel](https://openbabel.org) is a popular toolbox for chemical molecular modelling as it enables conversion, searching, filtering and analysis of chemical data." instead? I agree about removing the data storage aspect but I'm getting a bit nit-picky myself about the grammar.

Reply:
Absolutely fine by me, never trust me for grammar 😌 I switched to use _allow_ since _enable_ is already in the previous and next sentences but no big deal

Reply:
Easy! Do I need to checkout this PR and make that change on my end? I am not familiar with how we normally apply revisions? @cbouy @jennaswa

Reply:
In the code section of the PR you can click on the + next to a line to add a comment and there should be a button to create a suggestion there

Reply:
Since this was the only outstanding comment, I went ahead and included @lunamorrow's sentence, but switched out "enables" with "allows" - hope that is okay!

Reply:
> In the code section of the PR you can click on the + next to a line to add a comment and there should be a button to create a suggestion there

Thanks! Great to know for next time :)

> Since this was the only outstanding comment, I went ahead and included @lunamorrow's sentence, but switched out "enables" with "allows" - hope that is okay!

That is perfect, thank you for making that adjustment @jennaswa :)